code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -394,17 +395,47 @@
 	set category = "pAI Commands"
 	set name = "Choose Chassis"
 
+	var/list/my_choices = list()
 	var/choice
 	var/finalized = "No"
-	while(finalized == "No" && client)
 
-		choice = input(usr,"What would you like to use for your mobile chassis icon? This decision can only be made once.") as null|anything in possible_chassis
-		if(!choice) return
+	//check for custom_sprite
+	if(!custom_sprite)
+		var/file = file2text("config/custom_sprites.txt")
+		var/lines = splittext(file, "\n")
+
+		for(var/line in lines)
+		// split & clean up
+			var/list/Entry = splittext(line, ":")
+			for(var/i = 1 to Entry.len)
+				Entry[i] = trim(Entry[i])
+
+			if(Entry.len < 2 || Entry[1] != "pai")			//ignore incorrectly formatted entries or entries that aren't marked for pAI
+				continue
+
+			if(Entry[2] == ckey)							//They're in the list? Custom sprite time, var and icon change required
+				custom_sprite = 1
+				my_choices["Custom"] = "[ckey]-pai"

```

Comment:
So it is... must have been leftover from when I was initially thinking of adding support for multiple custom sprite sets on a single ckey (which I decided not to do in this PR) that I missed.